node_modules/grpc/index.d.ts - Imports are not permitted in module augmentations

### Problem description
Im using grpc with typescript but the typescript compile command fails with the following error: 'Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.'

### Reproduction steps
Install grpc in node project with typescript. Add a file that imports grpc. `run tsc -p tsconfig.json` wirh target=es6 and module=commonjs. 

### Environment
 - macOS Mojave
 - Node version 11.10
 - Node installation method : nvm
 - If applicable, compiler version [e.g. clang 3.8.0-2ubuntu4]
 - Package name and version:  gRPC@^1.23.2

### Additional context
Add any other context about the problem here. If possible, attach full logs. Do not try to omit anything for brevity, but instead include absolutely everything. Please try and set your operating system's locale to English, so that logs contain error messages in the English language.

### `$ tsc -p tsconfig.json`

```
node_modules/grpc/index.d.ts:21:3 - error TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.

21   import { Message, Service as ProtobufService } from "protobufjs";
     ~~~~~~

node_modules/grpc/index.d.ts:21:55 - error TS2307: Cannot find module 'protobufjs'.

21   import { Message, Service as ProtobufService } from "protobufjs";
                                                         ~~~~~~~~~~~~

node_modules/grpc/index.d.ts:22:3 - error TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.

22   import { Duplex, Readable, Writable } from "stream";
     ~~~~~~

node_modules/grpc/index.d.ts:23:3 - error TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.

23   import { SecureContext } from "tls";
     ~~~~~~


Found 4 errors.
```

### `$ cat tsconfig.json`

```
{
  "compilerOptions": {
    "target": "es2015",
    "module": "commonjs"
  }
}
```

### `$ cat package.json`
```
{
   ......(generated by npm init)

  "dependencies": {
    "grpc": "^1.23.2"
  },
  "devDependencies": {
    "@types/node": "^12.7.2"
  }
}
```

### `$ tsc -p tsconfig.json`

```
node_modules/grpc/index.d.ts:21:3 - error TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.

21   import { Message, Service as ProtobufService } from "protobufjs";
     ~~~~~~

node_modules/grpc/index.d.ts:21:55 - error TS2307: Cannot find module 'protobufjs'.

21   import { Message, Service as ProtobufService } from "protobufjs";
                                                         ~~~~~~~~~~~~

node_modules/grpc/index.d.ts:22:3 - error TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.

22   import { Duplex, Readable, Writable } from "stream";
     ~~~~~~

node_modules/grpc/index.d.ts:23:3 - error TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.

23   import { SecureContext } from "tls";
     ~~~~~~


Found 4 errors.
```

### `$ cat tsconfig.json`

```
{
  "compilerOptions": {
    "target": "es2015",
    "module": "commonjs"
  }
}
```

### `$ cat package.json`
```
{
   ......(generated by npm init)

  "dependencies": {
    "grpc": "^1.23.2"
  },
  "devDependencies": {
    "@types/node": "^12.7.2"
  }
}
```

I also experience this issue. 

Here is my output:

```
[11:44:49 AM] Starting compilation in watch mode...

node_modules/.registry.npmjs.org/grpc/1.23.2/node_modules/grpc/index.d.ts:21:3 - error TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.

21   import { Message, Service as ProtobufService } from "protobufjs";
     ~~~~~~

node_modules/.registry.npmjs.org/grpc/1.23.2/node_modules/grpc/index.d.ts:21:55 - error TS2307: Cannot find module 'protobufjs'.

21   import { Message, Service as ProtobufService } from "protobufjs";
                                                         ~~~~~~~~~~~~

node_modules/.registry.npmjs.org/grpc/1.23.2/node_modules/grpc/index.d.ts:22:3 - error TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.

22   import { Duplex, Readable, Writable } from "stream";
     ~~~~~~

node_modules/.registry.npmjs.org/grpc/1.23.2/node_modules/grpc/index.d.ts:23:3 - error TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.

23   import { SecureContext } from "tls";
     ~~~~~~

[11:44:51 AM] Found 4 errors. Watching for file changes.
```

I think it's not valid .d.ts. file because you can't use imports in declared module

--- a/packages/grpc-native-core/index.d.ts
+++ b/packages/grpc-native-core/index.d.ts
@@ -17,6 +17,7 @@
  */
 
 declare module "grpc" {
+  // add imports here, inside the "grpc" module, to keep it as an ambient module
   import { Message, Service as ProtobufService } from "protobufjs";
   import { EventEmitter } from "events";
   import { Duplex, Readable, Writable } from "stream";

--- a/packages/grpc-native-core/index.d.ts
+++ b/packages/grpc-native-core/index.d.ts
@@ -1,4 +1,3 @@
-import { EventEmitter } from "events";
 
 /*
  * Copyright 2019 gRPC authors.
@@ -19,6 +18,7 @@ import { EventEmitter } from "events";
 
 declare module "grpc" {
   import { Message, Service as ProtobufService } from "protobufjs";
+  import { EventEmitter } from "events";
   import { Duplex, Readable, Writable } from "stream";
   import { SecureContext } from "tls";
 

